jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpTester.java

```diff
@@ -428,9 +428,9 @@ public boolean content(ByteBuffer ref)
         }
 
         @Override
-        public void badMessage(BadMessageException failure)
+        public void badMessage(HttpException failure)
         {
-            throw failure;
+            HttpException.rethrowRuntime(failure);
```

Comment:
See previous comment, I would prefer here: `throw HttpException.asRuntimeException(failure);`.
Or perhaps just the cast: `throw (RuntimeException)failure;` since all `HttpException`s are `RuntimeException`s.

Reply:
No, that would be wrong, as `Error`s are only thrown by the JVM runtime, not applications.